.circleci/config.yml

```diff
@@ -435,6 +435,38 @@ jobs:
           name: Setup Elasticsearch Index Settings
           command: |
             ./web-api/setup-elasticsearch-index.sh $ENV
+      - run:
+          name: Admin User Setup
+          command: |
+            if [ "${CIRCLE_BRANCH}" != "prod" ]; then
+              node shared/admin-tools/user/setup-admin.js
+            else
+              echo "skipping…"
+            fi
+      - run:
+          name: Test Users Setup
+          command: |
+            if [ "${CIRCLE_BRANCH}" != "prod" ]; then
+              ./shared/admin-tools/user/setup-test-users.sh $ENV
+            else
+              echo "skipping…"
+            fi
+      - run:
+          name: Setup Judge Users File Var
+          command: |
+            if [ "${CIRCLE_BRANCH}" != "prod" ] && [ "${CIRCLE_BRANCH}" != "test" ] && [ "${CIRCLE_BRANCH}" != "migration" ]; then
```

Comment:
What about ustc:staging?

Reply:
Good question. I was thinking only test and migration were prod-like, but of course staging is. Is migration?

Reply:
I believe glue jobs are also performed to staging but I could be wrong. @jimlerza is that right?

Reply:
Glue jobs are only performed to migration and test, although I think it's worth checking staging's Cognito pool to make sure the "real" judge accounts don't already exist there.

Reply:
Confirmed - the only judge users in staging's Cognito pool are of the `@example.com` variety, so that is an environment where we can/should import judges.